src/main/webapp/WEB-INF/views/customerSearchScreen.html

```diff
@@ -0,0 +1,146 @@
+
```

Comment:
How about renaming the file to be customerServiceSearch.html instead? Might help avoid confusion later.